tests/test_aperture.py

```diff
@@ -51,6 +52,25 @@ def test_regular_polygon_aperture():
 				circum_diameter=pupil_diameter, num_sides=num_sides, angle=angle
 			)
 
+def test_circular_aperture():
+	pupil_diameter = 1
+
+	for diameter in [1, 0.5]:
+		name = 'circular/pupil'
+		name += '_small' if diameter < 0.7 else '_large'
+
+		# Use a functools.partial() here to avoid the name collision of the diameter argument.
+		check_aperture(functools.partial(circular_aperture, diameter=diameter), pupil_diameter, name)
+
+def test_rectangular_aperture():
+	pupil_diameter = 1
+
+	for size in [[0.5, 0.5], [1, 0.5]]:
+		name = 'rectangular/pupil'
+		name += '_square' if size[0] == size[1] else '_elongated'
+
+		check_aperture(rectangular_aperture, pupil_diameter, name, size=size)
```

Comment:
rectangular_aperture will become make_rectangular_aperture due to PR #141 .

Reply:
I know, which is why #141 is a draft PR and will not be merged until this PR. Any merge conflicts will be fixed in #141, not the other way around.